frontend/src/components/Item/ItemMenu.vue

```diff
@@ -244,10 +244,10 @@ function getLibraryOptions(): MenuOption[] {
             useSnackbar(t('libraryRefreshQueued'), 'normal');
             taskManager.startTask({
               type: TaskType.LibraryRefresh,
-              id: menuProps.item.Id,
-              data: menuProps.item.Name,
+              id: menuProps.item.Id as string,
+              data: menuProps.item.Name as string,
```

Comment:
```suggestion
              id: menuProps.item.Id || '',
              data: menuProps.item.Name || '',
```

This is what we're doing in other places, and let us be sure this is really 100% safe at runtime.